appcues/src/main/java/com/appcues/debugger/DebuggerViewModel.kt

```diff
@@ -212,16 +203,19 @@ internal class DebuggerViewModel(
         }
     }
 
-    fun onBackPress() {
+    fun closeExpandedView() {
```

Comment:
there were also multiple functions doing the same thing here - returning from `Expanded` state and going to `Idle` - so now they share this one function.